SumoLogic.Logging.NLog/BufferedSumoLogicTarget.cs

```diff
@@ -254,6 +272,8 @@ protected override void InitializeTarget()
             this.SumoLogicMessageSender.RetryInterval = TimeSpan.FromMilliseconds(this.RetryInterval);
             this.SumoLogicMessageSender.ConnectionTimeout = TimeSpan.FromMilliseconds(this.ConnectionTimeout);
             this.SumoLogicMessageSender.Url = string.IsNullOrEmpty(this.Url) ? null : new Uri(this.Url);
+            this.SumoLogicMessageSender.SourceCategory = string.IsNullOrEmpty(this.SourceCategory) ? null : this.SourceCategory;

```

Comment:
You should probably guard against whitespace as well. `string.IsNullOrWhitespace`